backend/src/main/java/com/woowacourse/momo/service/schedule/ScheduleService.java

```diff
@@ -75,6 +76,22 @@ private Schedule createSchedule(Meeting meeting, Attendee attendee, AvailableDat
         return new Schedule(attendee, availableDate, timeslot);
     }
 
+    @Transactional(readOnly = true)
+    public SchedulesResponse findAllSchedules(String uuid) {
+        Meeting meeting = meetingRepository.findByUuid(uuid)
+                .orElseThrow(() -> new MomoException(MeetingErrorCode.NOT_FOUND_MEETING));
+        List<Attendee> attendees = attendeeRepository.findAllByMeeting(meeting);
+        List<Schedule> schedules = findAllSchedules(attendees);
```

Comment:
```suggestion
        List<Schedule> schedules = scheduleRepository.findAllByAttendeeIn(attendees);
```

Stream안에서 단일 select 쿼리를 날리는 것 보다 In 절에 한 번에 조회하는게 더 효율적일 것 같네요.

Reply:
좋은 방법이네요!  수정했습니다~